.github/workflows/release-create.yml

```diff
@@ -96,6 +96,7 @@ jobs:
         VERSION_PATTERH="$(echo ${RELEASE_VERSION} | sed 's/\./\\./g' )"
         grep -q "^${VERSION_PATTERH} [(][0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9][)]$" < python/HISTORY.rst || FAILS="${FAILS} python/HISTORY.rst"
         grep -q "^## ${VERSION_PATTERH} Release [(][A-Z][a-z]* [0-9][0-9]*, [0-9][0-9][0-9][0-9][)]$" < site/docs/try/releases.md || FAILS="${FAILS} site/docs/try/releases.md"
+        grep -q " ${VERSION_PATTERH} " < site/docs/try/server-upgrade.md || FAILS="${FAILS} site/docs/try/server-upgrade.md"
```

Comment:
you might want to update the comment in line 92 as well, since it only mentions history.rst/releases.md